tests/by-util/test_head.rs

```diff
@@ -378,3 +380,41 @@ fn test_presume_input_pipe_5_chars() {
         .run()
         .stdout_is_fixture("lorem_ipsum_5_chars.expected");
 }
+
+#[cfg(all(not(target_os = "windows"), not(target_os = "macos")))]
+fn run_and_compare_with_gnu_live_reference(ts: &TestScenario, args: &[&str]) {
```

Comment:
i don't think we need this
we can just check that we get a string longer than 1  char ?

Reply:
Hmm, I guess yes. But why not keeping the already existing more strict tests?

Reply:
because we already run the GNU testsuite in a separate test suite
and I don't want to make assumptions that the system has the GNU coreutils installed for our testsuite to work

Reply:
thanks for the explanations :-) I'll do it.

Reply:
thanks